libsql/src/connection.rs

```diff
@@ -25,6 +26,14 @@ pub(crate) trait Conn {
     fn last_insert_rowid(&self) -> i64;
 
     async fn reset(&self);
+
+    fn enable_load_extension(&self, _onoff: std::ffi::c_int) -> Result<()> {
+        Err(crate::Error::LoadExtensionNotSupported)
+    }
```

Comment:
can we make that a `bool`? It's odd to have FFI code leak that high up
```suggestion
    fn toggle_load_extension(&self, enabled: bool) -> Result<()> {
        Err(crate::Error::LoadExtensionNotSupported)
    }
```